x-pack/solutions/search/plugins/search_query_rules/public/components/empty_prompt/empty_prompt.tsx

```diff
@@ -0,0 +1,229 @@
+                        </EuiFlexItem>
+                      </EuiFlexGroup>
+                    </EuiFlexItem>
+                  </EuiFlexGroup>
+                  <EuiFlexGroup responsive={false} gutterSize="s" direction="row">
+                    <EuiFlexItem grow={false}>
+                      <EuiIcon type="check" />
+                    </EuiFlexItem>
+                    <EuiFlexItem grow={false}>
+                      <EuiFlexGroup responsive={false} gutterSize="xs" direction="column">
+                        <EuiFlexItem grow={false}>
+                          <EuiTitle size="xxs">
+                            <h5>
+                              <FormattedMessage
+                                id="xpack.queryRules.emptyPrompt.feature2.title"
+                                defaultMessage="Level up your search relevance"
+                              />
+                            </h5>
+                          </EuiTitle>
+                        </EuiFlexItem>
+                        <EuiFlexItem>
+                          <EuiText size="xs" color="subdued">
+                            <p>
+                              <FormattedMessage
+                                id="xpack.queryRules.emptyPrompt.feature2.description"
+                                defaultMessage="Expand upon your existing RAG or BM25 search results by including your query parameters to help influence your search relevance."
+                              />
+                            </p>
+                          </EuiText>
+                        </EuiFlexItem>
+                      </EuiFlexGroup>
+                    </EuiFlexItem>
+                  </EuiFlexGroup>
+                  <EuiFlexGroup responsive={false} gutterSize="s" direction="row">
+                    <EuiFlexItem grow={false}>
+                      <EuiIcon type="check" />
+                    </EuiFlexItem>
+                    <EuiFlexItem grow={false}>
+                      <EuiFlexGroup responsive={false} gutterSize="xs" direction="column">
+                        <EuiFlexItem grow={false}>
+                          <EuiTitle size="xxs">
+                            <h5>
+                              <FormattedMessage
+                                id="xpack.queryRules.emptyPrompt.feature3.title"
+                                defaultMessage="Collaborate across workflows"
+                              />
+                            </h5>
+                          </EuiTitle>
+                        </EuiFlexItem>
+                        <EuiFlexItem>
+                          <EuiText size="xs" color="subdued">
+                            <p>
+                              <FormattedMessage
+                                id="xpack.queryRules.emptyPrompt.feature3.description"
+                                defaultMessage="Update your query rule logic and export the updates to code for easy developer handoff."
```

Comment:
```suggestion
                                defaultMessage="Update query rules and export as code for seamless developer handoff."
```

Missted this straggler, it's fine already but just a bit longwinded

Reply:
I'd prefer to understand the workflow here to potentially improve this but I think the fact that you can export the query rules as code is clearly communicated. This copy implies the person working in the query rules UI has a different role, but it's not entirely clear from the rest of the copy that we're targeting a specific user-type here. I'm probably overthinking but....